instance/production_config.cfg

```diff
@@ -0,0 +1,31 @@
+import os
```

Comment:
What actually decides which config is being used? Don't we have to change our build process so that production_config.cfg will be used?